Gemfile

```diff
@@ -42,6 +42,7 @@ group :bench do
 end
 
 group :test do
+  gem 'rspec-rails'
```

Comment:
Anything required to run tests should be in gemspec. I am also inclined to think this should be its own gem.  I 'll help